src/main/java/picard/cmdline/CommandLineProgram.java

```diff
@@ -102,6 +105,12 @@
 
     @Option(doc="Google Genomics API client_secrets.json file path.", common = true)
     public String GA4GH_CLIENT_SECRETS="client_secrets.json";
+
+    @Option(shortName = "use_jdk_deflater", doc = "Use the JDK Deflater instead of the IntelDeflater for writing BAMs", common = true)
```

Comment:
This controls more than just writing BAMs -- it controls anything written via `BlockCompressedOutputStream` (applies to the second argument below as well)